src/main/java/book/store/controller/BookController.java

```diff
@@ -18,47 +24,70 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@OpenAPIDefinition(tags = {
+        @Tag(name = "Create"),
+        @Tag(name = "Update"),
+        @Tag(name = "Find"),
+        @Tag(name = "Search"),
+        @Tag(name = "Delete"),
+})
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "/books")
 public class BookController {
 
     private final BookService bookService;
 
-    @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
-    }
-
-    @GetMapping("/{id}")
-    public BookDto getBookById(@PathVariable Long id) {
-        return bookService.findById(id);
-    }
-
+    @Tag(name = "Create")
     @PostMapping
+    @Operation(summary = "Create new book", description = "Create new book")
     public BookDto createBook(@RequestBody @Valid CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
 
+    @Tag(name = "Update")
     @PutMapping("/{id}")
+    @Operation(summary = "Update book data", description = "Update book data by id")
     public BookDto updateBook(@PathVariable Long id, @RequestBody CreateBookRequestDto bookDto) {
         return bookService.updateBook(id, bookDto);
     }
 
-    @ResponseStatus(HttpStatus.OK)
+    @Tag(name = "Find")
+    @GetMapping
+    @Operation(summary = "Get all books", description = "Get a list of undeleted books "
+            + "with the possibility of sorting and pagination")
+    @PageableAsQueryParam
+    public List<BookDto> getAll(@Parameter(hidden = true) Pageable pageable) {
```

Comment:
Return Page<BookDto> instead of List<BookDto>:
```suggestion
    public Page<BookDto> getAll(@Parameter(hidden = true) Pageable pageable) {
```
This change will:
1. Allow Swagger to show pagination schema properly.
2. Return content, totalElements, totalPages, etc.